pkg/users/local.js

```diff
@@ -271,6 +272,30 @@ function is_user_in_group(user, group) {
     return false;
 }
 
+var AccountItem = React.createClass({
+    displayName: 'AccountItem',
+    click: function() {
+        cockpit.location.go([this.props.name]);
```

Comment:
this should only trigger when the primary button is used, otherwise we get strange behavior when someone wants to use the alternate mouse button to copy the link or inspect the page
https://github.com/cockpit-project/cockpit/blob/master/pkg/lib/cockpit-components-dialog.jsx#L80